randovania/server/multiplayer/world_api.py

```diff
@@ -350,6 +351,8 @@ def emit_world_pickups_update(sa: ServerApp, world: World):
                 {
                     "provider_name": action.provider.name,
                     "pickup": _base64_encode_pickup(pickup_target.pickup, resource_database),
+                    "location": action.location,
```

Comment:
Does providing the index when it's not co-op ever make sense? The world it's from is not included so you don't have access to the correct GameDescription.

Could instead merge both is_coop with location, making it say `coop_location` which is an `Pickup Index | None`

Reply:
I'm happier with this